zipline/src/jsMain/kotlin/app/cash/zipline/ziplineJs.kt

```diff
@@ -100,12 +100,12 @@ private class ZiplineJs : Zipline(), JsPlatform, CallChannel  {
     // Connect platforms using our newly-bootstrapped bridges.
     endpoint.set<JsPlatform>(
       name = "app.cash.zipline.jsPlatform",
-      serializersModule = ZiplineSerializersModule,
+      serializersModule = EmptySerializersModule,
       instance = this
     )
     hostPlatform = endpoint.get(
       name = "app.cash.zipline.hostPlatform",
-      serializersModule = ZiplineSerializersModule
+      serializersModule = EmptySerializersModule
```

Comment:
one PR I have on my queue – default parameter for `serializersModule`. It’s not implemented already because it interacts with the rewriter.